definitions/variable/variables.yaml

```diff
@@ -10550,3 +10550,423 @@
     description: Subsidies for energy infrastructure related to Electricity (transport
       and storage)
     unit: billion EUR2020/yr
+- Minerals|Aluminium|Wind:
+    description: Demand of Aluminium for Wind Power Plants
+    unit: Mt/yr
+- Minerals|Aluminium|PV:
+    description: Demand of Aluminium for PV Power Plants
+    unit: Mt/yr
+- Minerals|Aluminium|CSP:
+    description: Demand of Aluminium for CSP Plants
+    unit: Mt/yr
+- Minerals|Aluminium|Stationary Fuel Cells:
+    description: Demand of Aluminium for Stationary Fuel Cells
+    unit: Mt/yr
+- Minerals|Aluminium|Power Plants (renewable):
```

Comment:
If it's a thermal power plant, can you really distinguish the plants between fossil gas vs. biogas?